cmake/SearchForStuff.cmake

```diff
@@ -203,7 +203,6 @@ if(PULSEAUDIO_FOUND)
     include_directories(${PULSEAUDIO_INCLUDE_DIR})
 endif()
 
-
```

Comment:
Why remove this line?

Reply:
Extra blank lines are a pet peeve of mine and I removed it by habit when looking at the above lines. I can revert this change if you wish? Although it fits the style of the rest of the file more like this.

Reply:
No it's fine, I just wanted to know the reasonning, this is why I put it as a comment and not a request for changes